Makefile

```diff
@@ -76,11 +92,18 @@ prettier: npm
 
 all: hub sensor
 
+$(SENSOR_OBJ_DIR)/%.o: $(SENSOR_SRC_DIR)/%.cpp
+	$(CXX) $(CXXFLAGS) -c -o $@ $<
+
+$(SENSOR_BIN_DIR)/%: $(SENSOR_OBJ_DIR)/%.o
+	$(CXX) $(CXXFLAGS) -o $@ $^
+
 flash_sensor:
+	@echo "TODO Depends on having hardware."
 	$(SENSOR_MAKE) flash
 
-sensor:
-	$(SENSOR_MAKE) all
+
+sensor: $(SENSOR_BIN_FILES)
```

Comment:
This target is not correct anymore, I changed it to use the makefile provided by the processor board manufacturer which had a target to build. Was there a reason it was changed back?

Reply:
I added that back because that was there previously. I also added that back because you said yesterday i deleted it even though i dont how how it says i deleted it